_maps/configs/NT-C_clipper.json

```diff
@@ -0,0 +1,38 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"prefix": "NT-C",
+	"namelists": ["GENERAL", "BRITISH_NAVY", "MERCANTILE", "SPACE"],
+	"map_name": "Indispubable-class Secure Transport",
```

Comment:
Fix this

Reply:
fixed the typo, anything else?

Reply:
I haven't given it a proper look-through yet. Check your decals, make sure the ship is capable of docking, undocking, and moving on the overmap. And please, please, don't run piping through walls, it makes it significantly harder to fix in-game.